jvm-runtime/testdata/go/gomodule/server.go

```diff
@@ -280,3 +280,13 @@ func TypeEnumVerb(ctx context.Context, val AnimalWrapper) (AnimalWrapper, error)
 //func MixedEnumVerb(ctx context.Context, val Mixed) (Mixed, error) {
 //	return val, nil
 //}
+
+//ftl:verb export
+func PrimitiveResponseVerb(ctx context.Context, val string) (int, error) {
```

Comment:
So looking into this I don't know if we need this change:

Currently we generate: 

```
@VerbClientDefinition(
    name = "intVerb",
    module = "gomodule"
)
public interface IntVerbClient extends VerbClient<Long, Long> {
  long call(long value);
}
```

Which is valid, even though the primitive method does not override the interface type, because we actually generate both methods:

https://github.com/TBD54566975/ftl/blob/b0465e3c2d14cb72400b8724058db1445c663619/jvm-runtime/ftl-runtime/common/deployment/src/main/java/xyz/block/ftl/deployment/VerbProcessor.java#L93

Basically in the int -> int case we generate three different methods:

Object -> Object
Long -> Long
long -> long

Reply:
The compile error doesn't show up for `IntVerb` because the generated signature of `call(long)` is different to `call(Long)` in `VerbClient`. Need to have an Object param with primitive response value to make the signature the same but different response type.

I guess we can restrict boxing to cases where the param is an Object and the response isn't.

Reply:
Ah, I see what you mean. 

I wonder how much implementing `VerbClient` actually matters. For the generated classes I don't think it actually adds anything, and not being able to use primitives is just kinda annoying. I am not 100% happy with the way verb clients currently work anyway, as invocation within the same module is clunky. Let me think about this a bit.

Reply:
Possibly something like: https://github.com/TBD54566975/ftl/compare/stuartwdouglas/jvm-api-refactor?expand=1